jscomps/LoginManagerPrompter.js

```diff
@@ -1491,7 +1491,7 @@ LoginManagerPrompter.prototype = {
    *        The login captured from the form.
    */
   _showSaveLoginNotification : function (aBrowser, aLogin) {
-    var displayHost = this._getShortDisplayHost(aLogin.hostname);
```

Comment:
You can consider changing `_getShortDisplayHost()` because it's also used in `promptToChangePasswordWithUsernames()` and `_showChangeLoginNotification()`.

Reply:
Yes, I agree that we should have the same approach in all these similar cases. So I think we don't need this `_getShortDisplayHost` function anymore, because we want to show the full hostname, and we already have that full hostname in `aLogin.displayOrigin` property. So I deleted the function and replaced it's calls with assignment to the value of that property.